scripts/install-nix-from-closure.sh

```diff
@@ -30,7 +30,14 @@ if [ "$(uname -s)" = "Darwin" ]; then
     fi
 
     printf '\e[1;31mSwitching to the Multi-User Darwin Installer\e[0m\n'
-    exec "$self/install-darwin-multi-user"
+    exec "$self/install-multi-user"
+    exit 0
+fi
+
+# Linux & Systemd support
+if [ "$(uname -s)" = "Linux" ] && [ -e /run/systemd/system ]; then
```

Comment:
Imagine the egg on your face when systemd starts supporting darwin

Reply:
:sweat_smile: